source/install/install-tar.rst

```diff
@@ -1,8 +1,260 @@
+      <span>Copy to clipboard<span>
+    </button>
+
+  </div>
+
+Download an older release
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are looking for an older release, these can be found in our `version archive </upgrade/version-archive.html>`__ documentation.
+
+Install
+-------
+
+Install the Mattermost Server by extracting the tarball, creating users and groups, and setting file/folder permissions. 
+
+First extract the tarball:
+
+.. code-block:: none
+  :class: mm-code-block 
+
+    tar -xvzf mattermost*.gz
+
+Now move the entire folder to the ``/opt`` directory (or whatever path you require):
+
+.. code-block:: none
+  :class: mm-code-block 
+
+    sudo mv mattermost /opt
+
+.. note::
+
+	If you choose a custom path, ensure this alternate path is used in all steps that follow.
+
+By default the Mattermost Server uses ``/opt/mattermost/data`` as the folder for files. This can be changed in the System Console during setup (even using alternative storage such as S3). Create the default storage folder:
+
+.. code-block:: none
+  :class: mm-code-block 
+    
+    sudo mkdir /opt/mattermost/data
+
+Now set up a user and group called ``mattermost``:
+
+.. code-block:: none
+  :class: mm-code-block 
+    
+    sudo useradd --system --user-group mattermost
+
+.. note::
+
+	If you choose a custom user and group name, ensure it is used in all the steps that follow.
+
+Set the file and folder permissions for your installation:
+
+.. code-block:: none
+  :class: mm-code-block 
+    
+    sudo chown -R mattermost:mattermost /opt/mattermost
+
+Give the ``mattermost`` group write permissions to the application folder:
+
+.. code-block:: none
+  :class: mm-code-block 
+        
+    sudo chmod -R g+w /opt/mattermost
+
+You will now have the latest Mattermost Server version installed on your system. Managaging starting and stopping the Mattermost Server is done using ``systemd``. Create the systemd unit file:
+
+.. code-block:: none
+  :class: mm-code-block 
+    
+    sudo touch /lib/systemd/system/mattermost.service
+
+As root, edit the systemd unit file to add the following lines:
+
+.. code-block:: none
+  :class: mm-code-block 
+
+    [Unit]
+    Description=Mattermost
+    After=network.target
+
+    [Service]
+    Type=notify
+    ExecStart=/opt/mattermost/bin/mattermost
+    TimeoutStartSec=3600
+    KillMode=mixed
+    Restart=always
+    RestartSec=10
+    WorkingDirectory=/opt/mattermost
+    User=mattermost
+    Group=mattermost
+    LimitNOFILE=49152
+
+    [Install]
+    WantedBy=multi-user.target
+
+Save the file and reload systemd using ``sudo systemctl daemon-reload``. Mattermost Server is now installed and is ready for setup.
+
+.. note::
+	
+	If you are installing the Mattermost server on the same system as your database, you may want to add both ``After=postgresql.service`` and ``BindsTo=postgresql.service`` to the ``[Unit]`` section of the systemd unit file.
```

Comment:
@emdecr - Is it possible to stop the command in this note from being cut-off based on an arbitrary fixed width of the note panel?

Reply:
@cwarnermm Would the preferred formatting look as such?
<img width="1039" alt="Screen Shot 2023-04-27 at 12 33 44 PM" src="https://user-images.githubusercontent.com/10912003/234930079-26e12940-bdc3-470b-8b9c-26eb5431f25f.png">

Inline code elements have a `word-break: break-all;` CSS rule on them, but we can definitely remove.

@nab-77 - Can you screencap/comment on the sections of code you're referring to? Though, this may be the fix for them as well.

TY~

Reply:
Fantastic! Yes, the preferred formatting would keep commands and code intact as you've shown in your screenshot. Thank you!

Reply:
I'll review the generated preview and identify any additional text that would benefit from a CSS adjustment.

Reply:
There are no other unexpected text splits that we need to address in this PR.